### PR TITLE
chore: Clean up v11 upgrade guide

### PR DESCRIPTION
**Pull request by mannycarrera4**
## Summary

Fixes: https://github.com/Workday/canvas-kit/issues/2694

## Release Category
Documentation

---

### CODE REVIEW

File: modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx [hunk 1]
```diff
@@ -442,84 +418,89 @@ See below:
...
-identically as it did in previous versions.
+[styling API](https://workday.github.io/canvas-kit/?path=/docs/styling-basics--create-modifiers#createstyles-api).
+The React interface **has not changed**, but CSS variables are now used for dynamic properties.
 
-### Combobox
+The following components are affected: `PrimaryButton`, `SecondaryButton`,
+`TertiaryButton`,`DeleteButton`, `Card`, `CheckBox`, `Combobox`, `CountBadge`,
+`FormField (Preview)`, `Svg`, `SystemIcon`, `SystemIconCircle`, `AccentIcon`, `AppletIcon`,
+`Graphic`, `LoadingDots`, `Radio (Preview)`, `Select`, `StatusIndicator`, `Switch`, `Table` and
+`Text`
```
Comment (jaclynjessup) [inline modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx:462]: Is there a better way to have these listed out? Not sure this is the best format as it is hard to scan.

File: modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx
Comment (jaclynjessup) [inline modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx:1]: line 571 should be 
systemIconStyles has been deprecated and will be removed in a future version as a part of implementation of stencils and new tokens. Consider using systemIconStencil to get styles or class name.

**Review by josh-bagwell — APPROVED** [review]

File: modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx [hunk 2]
```diff
@@ -207,51 +151,61 @@ instead.
...
+- `Checkbox`
+- `DeleteButton`
+- `SecondaryButton`
+- `StatusIndicator`
+- `Switch`
+- `Table`
+- `TertiaryButton`
+- `TextArea`
+- `TextInput`
+- `Radio`
```
Comment (josh-bagwell) [inline modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx:208]:
```suggestion
- `Radio`
- `SecondaryButton`
- `StatusIndicator`
- `Switch`
- `Table`
- `TertiaryButton`
- `TextArea`
- `TextInput`
```

File: modules/styling/stories/examples/CSProp.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    maxWidth: '30rem',
+    backgroundColor: '#1E2329',
+  },
+});
+
+export const CSProp = () => {
+  return (
+    <Card cs={myStencil()}>
+      <Card.Heading cs={{color: cssVar(myStencil.vars.color)}}>The Future of Styling</Card.Heading>
+      <Card.Body cs={{color: '#fff'}}>
```
Comment (RayRedGoose) [inline modules/styling/stories/examples/CSProp.tsx:20]: Do we want to use raw hex values here?

Comment (mannycarrera4) [reply modules/styling/stories/examples/CSProp.tsx:20]: i can update to use tokens, i think i was testing!

File: modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx [hunk 4]
```diff
@@ -11,37 +13,51 @@ This guide contains an overview of the changes in Canvas Kit v11. Please
...
+  [new styling API](https://workday.github.io/canvas-kit/?path=/docs/styling-basics--create-stencil).
+  This shouldn't effect the way you currently style your components. Because we're moving away from
+  Emotion, support for style props will eventually be removed, however, this API provide backwards
+  compatability. If you want to slowly move off of Emotion as well, you can style styling components
+  via the `cs` prop or our new styling utilities.
+
+## Table of contents
+
 - [Codemod](#codemod)
-- [Component Promotions](#component-promotions)
```
Comment (josh-bagwell) [inline modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx:15]: Do we not want to call this out since this is a whole new component and not just a change?

Comment (mannycarrera4) [reply modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx:15]: In previous versions we've put this under `Component Updates` so I kept that way. Notice table is under component updates, and that's where the promotion is called out a long with any other updates to it!

File: modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx [hunk 5]
```diff
@@ -81,10 +97,10 @@ our team. We'd be very happy to walk you through the process to set you up for s
 
 ### Instructions
 
-The easiest way to run our codemod is to use `npx`.
+The easiest way to run our codemod is to use `npx` in your terminal.
 
 ```sh
-> npx @workday/canvas-kit-codemod v11 [path]
```
[the remark this answers is not in the document]
Comment (mannycarrera4) [reply modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx:87]: I just removed `>` character. That way when people copy past in their terminal it works.

[the remark this answers is not in the document]
Comment (mannycarrera4) [reply modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx:87]: ## Before

image

## After
image

[the remark this answers is not in the document]
Comment (josh-bagwell) [reply modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx:87]: Got it! Sounds good to me man!

File: modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx (hunk 4 above)
Comment (josh-bagwell) [reply modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx:15]: Perfect. I think that's a good way to go about it

File: modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx [hunk 6]
```diff
@@ -444,70 +454,95 @@ See below:
...
-identically as it did in previous versions.
+<StatusIndicator variant="red" emphasis="low">
+  <StatusIndicator.Label>Visual Breaking Change</StatusIndicator.Label>
+</StatusIndicator>
 
-### Combobox
+**PR:** [#2666](https://github.com/Workday/canvas-kit/pull/2666)
 
-**PR:** [#2570](https://github.com/Workday/canvas-kit/pull/2570)
+The `border` color for `SecondaryButton` on the `focus` state is now `transparent` and may cause a
```
[the remark this answers is not in the document]
Comment (mannycarrera4) [reply modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx:532]: I removed this section since it's covered by `Component Style Updates`

**Pull request merged**